### PR TITLE
Adds a description for the Gigadrill

### LINKED ISSUE #17531: Gigadrill inherits desc from basic chair, modestly unfitting
#### Description of issue
That's a gigadrill.
You sit in this. Either by will or force. 

#### Difference between expected and actual behavior
I expected a unique desc string.  

#### Steps to reproduce 
Spawn a /obj/structure/bed/chair/vehicle/gigadrill
Examine it.  

#### Length of time in which bug has been known to occur
Likely since it's been added.  

#### Server revision
Game: df099306b1f9b52aff62f9cb3862b2abcb170171 
Patches: 1f151d7a38ff2301172c5585b8ec3a972ca3efe6 
Current Infomational Settings: 
Protect Authority Roles From Tratior: 1

#### Issue bingo
- [x] Issue could be reproduced at least once
- [x] Issue could be reproduced by different players
- [ ] Issue could be reproduced in multiple rounds
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] [Couldn't find an existing issue about this](https://github.com/d3athrow/vgstation13/issues)

### PR DESCRIPTION
**Pull request by west3436**
## What this does

Restores the description for the gigadrill as first defined in the ancient PR #13639.
## Why it's good

Closes #17531.

## Changelog

:cl:
 * rscadd: The Gigadrill now has a unique description.

### CODE REVIEW

**Comment by nervere** [comment]
this NEEDS to be a gurren lagann reference

**Comment by west3436** [comment]
> this NEEDS to be a gurren lagann reference

Provide the wording, weeb

**Review by jwhitak — APPROVED** [review]

File: code/game/objects/structures/vehicles/gigadrill.dm [hunk 1]
```diff
@@ -7,6 +7,7 @@
 
 /obj/structure/bed/chair/vehicle/gigadrill
 	name = "gigadrill"
+	desc = "An old and dusty vehicle with a mining drill - structure composed of titanium-carbide alloy, with tip and drill lines edged in an alloy of diamond and plasma"
```
Comment (Kurfursten) [inline code/game/objects/structures/vehicles/gigadrill.dm]: Missing period at end


Also here's a quote for you

If there's a wall in our way then we smash it down! If there isn't a path, then we carve one ourselves!

Comment (west3436) [reply code/game/objects/structures/vehicles/gigadrill.dm]: Excellent, I'll update this tomorrow

Comment (west3436) [reply code/game/objects/structures/vehicles/gigadrill.dm]: Implemented

**Pull request merged**